libasn1compiler/asn1c_save.c

```diff
@@ -234,7 +234,7 @@ asn1c__save_example_mk_makefile(arg_t *arg, const asn1c_dep_chainset *deps,
 
     for(int i = 0; i < argc; i++)
         safe_fprintf(mkf, "%s%s", i ? " " : "", argv[i]);
-	safe_fprintf(mkf, "\n\n");
+    safe_fprintf(mkf, "\n\n");
```

Comment:
Would you be open to removing this change? It has no relation to improving the test coverage.

Reply:
While this particular one-liner doesn't affect the coverage - it still makes sense.

I recommend keeping it in, rather than dedicating a whole new PR just to fix this original identification break.